docs/source/models/supported_models.md

```diff
@@ -112,7 +112,7 @@ See [this page](#generative-models) for more information on how to use generativ
     - :code:`THUDM/chatglm2-6b`, :code:`THUDM/chatglm3-6b`, etc.
     - ✅︎
     - ✅︎
-  * - :code:`CohereForCausalLM`,:code:`Cohere2ForCausalLM`
+  * - :code:`CohereForCausalLM`, :code:`Cohere2ForCausalLM`
```

Comment:
it seems `:code:` is quite sensitive to spaces. maybe we can swith to simple `` in the future since we are using markdown now. cc @DarkLight1337